remoteit/desktop
# DESK-813 Clear connections removes from CLI but keeps config data in desktop

### Description



Clear connections removes from CLI but keeps config data in desktop

### Test plan



1. Sign in
2. Configure a connection with custom info
3. Click connect and disconnect
4. Goto connections page and clear the connection
5. Go back to the connection from the device list
6. The connection setting should still be set
7. You should see a revert icon in the upper right
8. Click the revert icon to restore the default settings

### Ticket



<https://remoteit.atlassian.net/browse/DESK-813>
